docs/proposals/old-timer.md

```diff
@@ -207,7 +207,7 @@ All computations, filtering, etc should be performed in the sink.  Oldtimer
 should only be composing queries.  Ergo, the feature set of Oldtimer must
 represent the lowest-common-denominator of features supported by the sinks.
 Oldtimer is meant to be an API for performing basic aggregations supported by
-all of the sinks, and is not meant to be a general purpose query tool.
+all of the sinks and is not meant to be a general purpose query tool.
```

Comment:
this comma was not incorrect, and the pause makes the sentence easier to parse, since it's easy to see the the last `and` is a clarification.